src/lib/familiar.ts

```diff
@@ -1,5 +1,12 @@
-import { Familiar } from "kolmafia";
-import { $familiar, $item, have } from "libram";
+import { Familiar, familiarWeight, numericModifier } from "kolmafia";
+import { $familiar, $item, have, get } from "libram";
+
+export function runsOrNCFamiliar(): Familiar {
+  if (freeRunsFamiliar() !== $familiar`none`) {
```

Comment:
Let's make just one call to `freeRunsFamiliar` instead of one in the if condition, and one in the if body.